hugo/layouts/sitemap.xml

```diff
@@ -0,0 +1,30 @@
+{{ printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\" ?>" | safeHTML }}
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
```

Comment:
This is Hugo's [official template](https://gohugo.io/templates/sitemap-template/), but I don't understand why it has this `printf` here instead of just plain text.

Reply:
I suspect in order to pipe it through `safeHTML` so it doesn't get stripped out, but I haven't double-checked.